From 5318fc6afb3e1928a3470476dbb8301038863e8b Mon Sep 17 00:00:00 2001
From: LuoXiaoTan <lxt@rock-chips.com>
Date: Tue, 21 Mar 2017 00:45:50 -0700
Subject: [PATCH] arm64: rockchip_defconfig: enable RT5651_TC358749

enable CONFIG_SND_SOC_ROCKCHIP_RT5651_TC358749
for HDMIIn function

Change-Id: I5c77b3f03d75ea1c51a89a2396ff6472554025a5
Signed-off-by: LuoXiaoTan <lxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 2a4a3190c90f..1ed81f6709ab 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -497,6 +497,7 @@ CONFIG_SND_SOC_ROCKCHIP=y
 CONFIG_SND_SOC_ROCKCHIP_SPDIF=y
 CONFIG_SND_SOC_ROCKCHIP_HDMI_DP=y
 CONFIG_SND_SOC_ROCKCHIP_CDNDP=y
+CONFIG_SND_SOC_ROCKCHIP_RT5651_TC358749=y
 CONFIG_SND_SOC_CX2072X=y
 CONFIG_SND_SOC_DW_HDMI_AUDIO=y
 CONFIG_SND_SOC_BT_SCO=y
-- 
2.35.3

